test/e2e/persistent_volumes.go

```diff
@@ -25,31 +25,23 @@ import (
 	"k8s.io/kubernetes/pkg/api/resource"
 	"k8s.io/kubernetes/pkg/api/testapi"
 	client "k8s.io/kubernetes/pkg/client/unversioned"
-	"time"
 )
 
 // Marked with [Skipped] to skip the test by default (see driver.go),
 // the test needs privileged containers, which are disabled by default.
 // Run the test with "go run hack/e2e.go ... --ginkgo.focus=PersistentVolume"
```

Comment:
Actually, no, PersistentVolume matches the "It" clause below, not the "Describe" clause.  Leaving as is.